### DISCUSSION TITLE
start menu only

### DISCUSSION THREAD

**Discussion by nuclead-gnome** [valinet/ExplorerPatcher#2799] [General]
Is it possible to release only the windows 10 start menu as a separate package? I don't use the rest of the functionality, but I really like your windows 10 implementation of the start menu.

**Comment by pyrates999** [comment]
EP is just using the windows 10 start menu code that Microsoft still includes in windows 11.

**Reply by nuclead-gnome** [reply]
that's why I want to have a separate utility with which you can activate only the start menu. after all, Explorer Patcher uses a set of such features. for the rest of the settings (file explorer, taskbar), I use a different program.

**Reply by pyrates999** [reply]
Source code is available.

**Reply by nuclead-gnome** [reply]
If I could independently allocate to a separate utility only what relates to the start menu, I probably wouldn't write here. Is it logical?

**Reply by pyrates999** [reply]
I see no reason for this when you could just install EP.

**Reply by nuclead-gnome** [reply]
you just seem to want to chat and have no one to talk to.
